### PR TITLE
Improve Singularity Tab UI

### PR DESCRIPTION
**Pull request by FlamemasterNXF**

### CODE REVIEW

**Review by KhafraDev — CHANGES_REQUESTED** [review]
1. `.idea` folder needs to be removed
2. `index.html` changes should be reverted
3. `.innerHTML` needs to go, manually create the elements and use `appendChild` (etc.) instead.

**Comment by FlamemasterNXF** [comment]
> 2. `index.html` changes should be reverted

The only changes I made there were to give the sing upgrades a class that gave them margins and to slightly reorganizes the sing upgrades. Where is the issue with those changes?

**Comment by KhafraDev** [comment]
> Where is the issue with those changes?

There's about 8.6 thousand changes due to an indentation change afaict

**Comment by FlamemasterNXF** [comment]
> > Where is the issue with those changes?
> 
> There's about 8.6 thousand changes due to an indentation change afaict

Oh geez, WebStorm must've reformatted it. I will fix that immediately.

**Comment by FlamemasterNXF** [comment]
@KhafraDev have the 8.6k changes gone away?

File: src/singularity.ts [hunk 1]
```diff
@@ -10,17 +10,18 @@ import { toOrdinal } from './Utility'
  *
  */
 export const updateSingularityStats = (): void => {
-    const str = `You are in the ${toOrdinal(player.singularityCount)} singularity, and have ${format(player.goldenQuarks,0,true)} golden quarks.
-                 Global Speed is divided by ${format(calculateSingularityDebuff('Global Speed'), 2, true)}.
+    const color = player.runelevels[6]>0?'green':'red'
```
Comment (KhafraDev) [inline src/singularity.ts:13]:
```suggestion
    const color = player.runelevels[6] > 0 ? 'green' : 'red';
```

File: src/singularity.ts [hunk 2]
```diff
@@ -75,16 +76,16 @@ export class SingularityUpgrade {
         const maxLevel = this.maxLevel === -1
             ? ''
             : `/${this.maxLevel}`;
+        const color = this.maxLevel === this.level?'plum':'white'
```
Comment (KhafraDev) [inline src/singularity.ts:79]:
```suggestion
        const color = this.maxLevel === this.level ? 'plum' : 'white';
```

**Review by KhafraDev — CHANGES_REQUESTED** [review]
also needs a rebase (innerhtml changes are fine for now)

**Comment by FlamemasterNXF** [comment]
What commit am I rebasing to? (sorry if this is obvious, I don't use rebase often)

**Comment by KhafraDev** [comment]
15ac1cc98794069ebc1433151326a87ee738759f

**Review by KhafraDev — APPROVED** [review]

**Comment by Pseudonian** [comment]
LGTM

**Pull request merged**